data/pigui/modules/time-window.lua

```diff
@@ -0,0 +1,65 @@
+local Engine = import('Engine')
+local Game = import('Game')
+local ui = import('pigui/pigui.lua')
+local Vector = import('Vector')
+local Color = import('Color')
+local Lang = import("Lang")
+local lc = Lang.GetResource("core");
+local lui = Lang.GetResource("ui-core");
+local utils = import("utils")
+local Event = import("Event")
+
+local player = nil
+local pionillium = ui.fonts.pionillium
+local colors = ui.theme.colors
+local icons = ui.theme.icons
+-- names of the keys in lang/core/
+local months = {"MONTH_JAN", "MONTH_FEB", "MONTH_MAR", "MONTH_APR", "MONTH_MAY", "MONTH_JUN", "MONTH_JUL", "MONTH_AUG", "MONTH_SEP", "MONTH_OCT", "MONTH_NOV", "MONTH_DEC"}
+
+local button_size = Vector(32,32)
+local frame_padding = 3
+local bg_color = colors.buttonBlue
+local fg_color = colors.white
+
+
+local function displayTimeWindow()
+	player = Game.player
+	local year, month_num, day, hour, minute, second = Game.GetDateTime()
+	local month = lc[months[month_num]]
+	local date = string.format("%04i %s %i - %02i:%02i:%02i", year, month, day, hour, minute, second)
+	local current = Game.GetTimeAcceleration()
+	local requested = Game.GetRequestedTimeAcceleration()
+	function accelButton(name, tooltip)
+		local color = bg_color
+		if requested == name and current ~= name then
+			color = colors.white
+			tooltip = 'Requested ' .. tooltip
```

Comment:
Thanks for the catch, I supposed this was going away anyway, and didn't look at it again.
I'll fix it.